srcpkgs/OpenCoarrays/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'OpenCoarrays'
+pkgname=OpenCoarrays
+version=2.9.2
+revision=1
+build_style=cmake
+short_desc="OpenCoarray fortran PAGS paralellization library"
+maintainer="Alberto Ramos <alberto.ramos@ific.uv.es>"
+hostdepends="gcc gcc-fortran openmpi openmpi-devel cmake"
+makedepends="gcc gcc-fortran openmpi openmpi-devel cmake"
+depends="gcc gcc-fortran openmpi openmpi-devel cmake"
+license="GPL-3.0-or-later"
+homepage="http://www.opencoarrays.org/"
+distfiles="https://github.com/sourceryinstitute/OpenCoarrays/releases/download/2.9.2/OpenCoarrays-${version}.tar.gz"
```

Comment:
You hardcoded `2.9.2` in one spot in the URL yet expanded `${version}` in another? It's recommended to expand `${version}` when possible to make maintaining easier. Try switching `distfiles` to `distfiles="https://github.com/sourceryinstitute/OpenCoarrays/archive/${version}.tar.gz"` and match the checksum.